engines/sci/graphics/font.cpp

```diff
@@ -87,13 +91,16 @@ void GfxFontFromResource::draw(uint16 chr, int16 top, int16 left, byte color, bo
 	int y = 0;
 	int16 greyedTop = top;
 
-	byte *pIn = getCharData(chr);
+	SciSpan<const byte> charData = getCharData(chr);
+	if (!charData) {
+		error("Character '%c' does not exist in font %d", chr, _resourceId);
+	}
```

Comment:
I don’t understand. `getCharData` previously returned a null pointer if the requested character was out of range, not a pointer to the first character. So as far as I can tell, this is the same as it was before, except without the potential of a null pointer dereference?